lib/java/test-utils/src/main/java/org/enso/test/utils/ContextRule.java

```diff
@@ -0,0 +1,209 @@
+package org.enso.test.utils;
+
+import java.io.ByteArrayOutputStream;
+import java.util.Set;
+import java.util.concurrent.Callable;
+import java.util.function.Supplier;
+import org.enso.interpreter.runtime.EnsoContext;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Engine;
+import org.graalvm.polyglot.Source;
+import org.graalvm.polyglot.Value;
+import org.junit.rules.TestRule;
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+
+/**
+ * Test rule that ensures that {@link Context} is initialized and discarded in tests. If used as
+ * {@link org.junit.ClassRule}, {@link Context} will be initialized just once for the whole test
+ * class. If used as {@link org.junit.Rule}, a new {@link Context} will be initialized for each test
+ * method.
+ *
+ * <p>This class simply delegates most of the methods either directly to {@link Context} or to
+ * {@link ContextUtils}.
+ *
+ * <p>Note that {@link ContextRule} cannot be used inside methods annotated with {@link
+ * org.junit.runners.Parameterized.Parameters}.
+ */
+public final class ContextRule implements TestRule {
```

Comment:
Should this class have `Rule` suffix? `TemporaryFolder` is also a `@Rule` and it doesn't have a `Rule` suffix...

Reply:
Merged `ContextRule` into `ContextUtils` in 952b6752b2254a65fe63107c919021824066cb6c